prismlauncher.tera

```diff
@@ -0,0 +1,52 @@
+---
+whiskers:
+  version: 2.1.0
+  matrix:
+    - variant: ["theme.json", "themeStyle.css", "theme.json.license"]
+    - flavor
+    - accent
+  filename: "themes/{{flavor.identifier}}/{{accent}}/{{variant}}"
+tooltip_color: "dee5fc"
```

Comment:
Why is this set separately? And it isn't a Catppuccin color?

Reply:
I thought it was a Catppuccin color, but I checked https://catppuccin.com/palette and couldn't find it. Also, strangely, the theme currently uses the exact same color in all four flavors.

I figured setting it as a frontmatter variable was better than repeating it twice in the template. Also, this way its purpose is clearer and it is more obvious the template relies on an arbitrary hardcoded value since it is declared up front.

Reply:
Where is it used? Like the color of tooltips? That should just be `text`. If it is the background of tooltips, it should be `crust` or `mantle`.

Reply:
It appears to be used in the background color and text color of tooltips. I have replaced it with `crust` and `text` respectively.